docs/ocis/development/testing.md

```diff
@@ -376,6 +376,48 @@ BEHAT_FEATURE="tests/acceptance/features/apiNotification/emailNotification.featu
 make test-acceptance-api
 ```
 
+## Running Test Suite With Tika Service (@tikaServiceNeeded)
```

Comment:
docs for while using the local docker-compose setup for tests (see sections above)

Reply:
@saw-jan sorry i did not get you ? section above?

Reply:
we also need docs for running search tests using docker compose setup. Docs for compose setup are in the above sections